CoreLayersView/nbproject/project.properties

```diff
@@ -1,2 +1,2 @@
-javac.source=1.8
+javac.source=11
```

Comment:
This would have been because this work started whilst we were still on Java 8. Greatly appreciate your review @cygnus-x-1 :heart: